backend/oasst_backend/user_repository.py

```diff
@@ -62,3 +62,32 @@ def get_user_leaderboard(self, role: str) -> LeaderboardStats:
         ]
 
         return LeaderboardStats(leaderboard=result)
+
+    def query_users(
+        self,
+        limit: Optional[int] = 20,
+        ge: Optional[str] = None,
+        lt: Optional[str] = None,
+        auth_method: Optional[str] = None,
+    ) -> list[User]:
+
+        users = self.db.query(User)
+
+        if not self.api_client.trusted:
+            users = users.filter(User.api_client_id == self.api_client.id)
+
+        if auth_method:
+            users = users.filter(User.auth_method == auth_method)
+
+        if ge:
+            users = users.filter(User.display_name >= ge)
```

Comment:
Which is the display name of a User? I think that is a string, so how is comparing a string greater than ge?

Reply:
that is indeed possible and intended here: https://www.educba.com/postgresql-compare-strings/

Reply:
Strings can be compared in alphabetical order and PostgreSQL supports string comparison operators. [https://stackoverflow.com/questions/40601324/comparing-strings-in-postgres-using-comparison-operators](url)

Reply:
So we are comparing alphabetically? The display_name creation does follow an order when created?

Reply:
Yes, we are comparing alphabetically. I'm not sure what you mean by following an order when created

Reply:
I mean I do not understand which could be the use case in which we might want to compare alphabetically.

Reply:
We're also trying to figure it out in the main thread :)